### PR TITLE
Added etc/ci/performance downloads to .gitignore.

### PR DESCRIPTION
**Pull request by asajeffrey**
Added the files downloaded for `etc/ci/performance` to `.gitignore`, which gets rid of lot of spurious reporting from `git status`.

r? @shinglyu 

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes do not require tests because it's about git

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13248)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@
 /ports/android/obj
 /python/_virtualenv
 /python/tidy/servo_tidy.egg-info
+/etc/ci/performance/page_load_test
```
Comment (shinglyu) [inline .gitignore]: How about adding this to the `etc/ci/performance/.gitignore` instead? If someday we want to move the `etc/ci/performance` folder we don't need to remember to edit this top level gitignore.

Comment (asajeffrey) [inline .gitignore]: Done.

**Comment by shinglyu** [comment]
I don't believe the Travis error are related to this patch. @bors-servo r+

**Pull request merged**